src/data/resolvers/colonyActions.ts

```diff
@@ -242,6 +242,7 @@ export const colonyActionsResolvers = ({
           domainColor: null,
           blockNumber,
           motionState: null,
+          motionDomain: null,
```

Comment:
I approve of this fix, but it's not complete. 

To be complete you need to add both `motionState` and `motionDomain` default values to the case where we don't have a `transactionReceipt`.

Like I've did here: https://github.com/JoinColony/colonyDapp/blob/704ce13611d8d2d73daea8645957723c48f6b2c4/src/data/resolvers/colonyActions.ts#L290-L291

Or you can just leave it like this, since when my PR will be merged it will be fixed.